sauna-app/components/map.tsx

```diff
@@ -0,0 +1,61 @@
+import 'leaflet/dist/leaflet.css';
+import { MapContainer,  TileLayer, Marker, Popup} from 'react-leaflet'
+import L from 'leaflet'
+import style from '../styles/Home.module.css'
+import { useState, useEffect } from 'react';
+import { SaunaData } from '../pages/typeDefinitions'
+
+
+
+function Map() {
+
+  const [saunas, setSaunas] = useState<SaunaData>();
+
+  useEffect(() => {
+    const fetchAPI = async () => {
+      await fetch('http://localhost:3000/api/saunas')
+      .then((response) => response.json())
+      .then((data:SaunaData) => setSaunas(data))
+    }
+    fetchAPI()
+  }, [])
+
+
+  return (
+    <MapContainer className={style.map} center={[60.257215, 24.966394]} zoom={10} scrollWheelZoom={false}>
+      <TileLayer
+        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
```

Comment:
A question: Where this url comes from and what does it do? Because it seems that is should be a string literal with variables `s`, `x`, `y` and `z` 🤔

Reply:
Hmm not exactly sure about that url, I just copy pasted that from React Leaflet component, see this example: [https://react-leaflet.js.org/](https://react-leaflet.js.org/)

Reply:
It seems that there's an explanation: [https://leafletjs.com/reference.html#tilelayer](https://leafletjs.com/reference.html#tilelayer) so I guess it's okay as is 😄 I have worked with Leaflet before, but it's a long time ago.